### DISCUSSION TITLE
Move cache files to cache library on MacOS

### DISCUSSION THREAD

**Discussion by opterios** [JanDeDobbeleer/oh-my-posh#5439] [Troubleshoot] [answered]
I'm using oh-my-posh on a MacOS and I've set it up via zsh, macOS Sonoma, ver.14.6.1.

Everything is working perfectly, however I'm noticing that the omp.cache files are being stored on my user home directory. Is there any way I can set oh-my-posh up to place these files in the default OS .cache library?

![image](https://github.com/user-attachments/assets/1dda8b5e-e865-46bc-8241-b2d66706a42c)

**Comment by JanDeDobbeleer** [comment, marked as answer]
@opterios we do validate existence if the .cache dir, but you can set the `OMP_CACHE_DIR` environment variable to the correct folder.
